common/traits/wc_race_traits.txt

```diff
@@ -718,6 +718,66 @@ creature_lich = {
 	random = no
```

Comment:
No loc.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79702625-5ed70f80-82b7-11ea-978b-aada2784f8d7.png)

</details>

Reply:
re-evaluate whether it still appears